src/main/java/microsoft/exchange/webservices/data/core/EwsUtilities.java

```diff
@@ -381,8 +332,7 @@ public static boolean stringEquals(String first, String second) {
    * @param message   The message to use if assertion fails.
    */
   public static void EwsAssert(boolean condition, String caller, String message) {
-    assert condition : String.format("[%s] %s",
-        caller, message);
+    assert condition : String.format("[%s] %s", caller, message);
```

Comment:
I think we should get rid of basic java assert since this can be disabled and therefore should not be used in production code.